guides/common/modules/proc_reclaiming-space-from-on-demand-repositories.adoc

```diff
@@ -4,6 +4,13 @@
 If you set the _download policy_ to on demand, {Project} downloads packages only when the clients request them.
 You can clean up these packages to reclaim space.
 
+[NOTE]
+====
+Space reclamation requires an existing repository. For deleted repositories, wait for the next scheduled orphan cleanup or run: 
```

Comment:
We shouldn't use incomplete introductory sentence like `[...] run:`. This can cause problems for translations (for example because translation software might only display individual sentences, in which case you don't know what follows after `[...] run:`).

It's preferable to actually describe what the command does. From the users' perspective, it's also more helpful to know what you're actually asking them to do (the command as such might be completely unknown to them). So how about something like this: 

> For deleted repositories, wait for the next scheduled orphan cleanup or <do_XYZ> manually:"

Obviously, you should replace <do_XYZ> with a suitable description of what the command does. This is an excerpt from CLI:
> ~] # foreman-rake -T | grep "delete_orphaned_content"
> 
> rake katello:delete_orphaned_content                                                 # Remove orphaned and unneeded content/repos from a smart proxy

And the code for the same foreman-rake task: https://github.com/Katello/katello/blob/master/lib/katello/tasks/delete_orphaned_content.rake

Please try to use this information to update the instructions.